app_to_web.md

```diff
@@ -76,6 +76,53 @@ For a site to enable App<->Web attribution after integrating with the Web API, t
 
 A reporting origin responding with the `Attribution-Reporting-Register-OS-Source` or `Attribution-Reporting-Register-OS-Trigger` headers while `Attribution-Reporting-Support` does not contain `os` will cause the entire registration to fail (including web registration).
 
+## Optional: debugging reports
+
+The Cross App and Web Attribution Measurement is new and fairly complex. A successful attribution requires both the
+browser and OS-level support. As such, we are open to introducing a mechanism to
+learn more information about OS registrations. This ensures that the API can be
+better understood and help flush out any bugs (either in browser or OS or caller
+code).
+
+The reporting origins may opt in to receiving debugging reports by adding a new
+boolean [parameter](https://httpwg.org/specs/rfc8941.html#param) `debug-reporting` to the
+[items](https://httpwg.org/specs/rfc8941.html#item) in the
+`Attribution-Reporting-Register-OS-Source` and
+`Attribution-Reporting-Register-OS-Trigger` headers:
+
+```http
+Attribution-Reporting-Register-OS-Source: "https://adtech.example/register"; debug-reporting, "https://other-adtech.example/register"
+```
+
+```http
+Attribution-Reporting-Register-OS-Trigger: "https://adtech.example/register", "https://other-adtech.example/register"; debug-reporting
+```
+
+The `debug-reporting` [parameter](https://httpwg.org/specs/rfc8941.html#param)
+is set for each registration URL independently. When the OS registration is
+successfully delegated to the OS, regardless the result of the native OS
+attribution API, the browser will send a non-credentialed HTTP POST
+request to the registration URL's origin at the path
```

Comment:
Consider rephrasing: 
When the OS registration is successfully delegated to the OS, the browser will send a non-credentialed HTTP POST request to the registration URL's origin regardless of the native OS attribution API results.

Reply:
Thanks, updated slightly.